src/config/index.ts

```diff
@@ -17,6 +17,7 @@ export const PIWIK_SITE_ID = getEnvironmentValue("PIWIK_SITE_ID");
 export const PIWIK_START_GOAL_ID = getEnvironmentValue("PIWIK_START_GOAL_ID");
 export const PAYMENT_FEE = getEnvironmentValue("PAYMENT_FEE");
 export const LANDING_PAGE_URL = getEnvironmentValue("LANDING_PAGE_URL");
+export const LANDING_PAGE_STARTING_NEW_URL = getEnvironmentValue("LANDING_PAGE_STARTING_NEW_URL");
```

Comment:
Probably we need to add a news story to remove the LANDING_PAGE_URL and update [here](https://github.com/companieshouse/overseas-entities-web/blob/9bab1a6c7fe9b2ca1cfec15e5d8fc76767ebc3f8/views/layout.html#L57)